iotilecore/RELEASE.md

```diff
@@ -2,6 +2,14 @@
 
 All major changes in each released version of `iotile-core` are listed here.
 
+## HEAD
+
+## 5.0.10
```

Comment:
```suggestion
```

If you just remove the 5.0.10 here, since we're going to be putting everything in HEAD and then cutting a release after we merge your other security code.

I can't commit to your fork, so I can't apply this suggestion (There is a way I can do it, but it means I have to change the target branch, merge it there, then add my own commit and merge it in to master).

You should be able to:

1. Apply the suggestion
2. Update the branch (if out of date with master)
3. Squash and merge

If the test is hanging/failing I will override the check tomorrow or whenever I see the notification if it happens earlier.

Sorry for leaving this one hanging for so long. It seems like it fell through the cracks a bit. We'll do better next time!

Reply:
It'll be easier for me to merge these PRs and fix it in master